apps/next/src/pages/Onboarding/flows/ConnectLedgerFlow/ConnectLedgerFlow.tsx

```diff
@@ -41,12 +42,19 @@ export const ConnectLedgerFlow = () => {
         <ProvideWalletDetailsScreen
           step={3}
```

Comment:
I really liked the approach with the steps in `apps/next/src/pages/Onboarding/flows/ConnectKeystoneFlow/ConnectKeystoneFlow.tsx` maybe that could be used here as well
\#niceToHave

Reply:
It may be better to switch to the approach I've introduced in the Ledger PR:

<img width="515" height="320" alt="image" src="https://github.com/user-attachments/assets/d85050f9-b85c-461f-9d86-349219d5979a" />

Or may be not.
Either way, I don't think it's a pressing issue at the moment 🙏

Reply:
yea it really more like a nice to have